src/argparse_utils.h

```diff
@@ -26,6 +26,7 @@
 bool parse_common_args(const std::string& program,
                        const cxxopts::Options& opts,
                        const cxxopts::ParseResult& result,
+                       boost::property_tree::ptree& conf,
```

Comment:
this is the main change. the exe's give an uninitialized one in here and we populate it. defining a local `conf` and changing it here makes no sense

Reply:
further down we add `mjolnir.concurrency` derived from the `-j` flag, the `mjolnir.concurrency` flag or whatever the hardware has